### ISSUE TITLE
Rerunning tests causes them to fail because the container is not cleaned up.

### ISSUE THREAD

**Issue by santiwanti** [testcontainers/testcontainers-rs#733]
I'm using the testcontainers-module Postgres container and all tests pass on the first run, but on the second run they fail because when trying to add the rows, the elements already exist causing problems with UNIQUE constraints. Is it expected behavior that the tests make permanent changes to the environment? I thought the whole point of testcontainers was that each test ran in isolation and the environment would be a clean one for each test.

**Comment by DDtKey** [comment]
No, this isn't the expected behavior. Each new container starts an independent instance (until any customizations are made - like mounting postgres data or reusing the same container)

Could you share the code that causes the issue? How do you connect to postgres?
